paddle/framework/library_type.h

```diff
@@ -20,7 +20,13 @@ namespace framework {
 // For more details about the design of LibraryType, Please refer to
 // https://github.com/PaddlePaddle/Paddle/blob/develop/doc/design/operator_kernel_type.md#library
 
-enum class LibraryType { kPlain = 0, kMKLDNN = 1, kCUDNN = 2 };
+enum class LibraryType {
+  kPlain = 0,
+  kMKLDNN = 1,
+  kCUDNN = 2,
+  kCPU = 3,
```

Comment:
Here kPlain means the default library both in CPU and CUDA. Since we already have CPUPlace and CUDAPlace, we have no need to add kCPU library and kCUDA library. <CPUPlace, kPlain> and <CUDAPlace, kPlain> are the most common Operator kernels.

Reply:
make sense. done.